multiuser/permission/che-multiuser-permission-workspace/src/main/java/org/eclipse/che/multiuser/permission/workspace/server/spi/jpa/MultiuserJpaWorkspaceDao.java

```diff
@@ -311,8 +315,35 @@ public void onCascadeEvent(BeforeAccountRemovedEvent event) throws Exception {
               (maxItems, skipCount) ->
                   workspaceManager.getByNamespace(
                       event.getAccount().getName(), false, maxItems, skipCount))) {
+        // Will try to stop running/starting workspaces.
+        // At the moment is impossible to force stop for  STOPPING workspace
+        if (RUNNING.equals(workspace.getStatus()) || STARTING.equals(workspace.getStatus())) {
```

Comment:
@vparfonov wdyt, if we should we handle the `STOPPING` case somehow?

Reply:
Will throw Conflict Exception